docs/caching.mdx

```diff
@@ -0,0 +1,127 @@
+---
+sidebar_position: 6
+---
+
+import Link from '../src/components/link/Link';
+
+# Caching
+
+## Development Environments
+
+For all development purposes, we use a local memory-based cache `Caffeine Cache` which is automatically created during the server startup.
+And when the service stops, all data in the local cache will be automatically cleared.
+
+:::danger Caffeine Cache is only for development environments
+The Caffeine Cache is purely for development and test purposes only and must be avoided for production installation of xpanse.
+:::
+
+## Production Environments
+
+For production environments, we use a distributed cache `Redis Cache`. By **support**, we mean configurations for the cache added and also all
```

Comment:
For production environments, we use a distributed cache. The below distributed caches are at the moment fully supported. By **support**, we mean configurations for the cache added and also all functionalities tested.